docs/API/API-socket-options.md

```diff
@@ -1068,20 +1070,25 @@ undefined behavior:
 
   - **Bidirectional in Caller-Listener arrangement**: it is recommended to use
   a rule whereby you will be setting the `PBKEYLEN` exclusively either on the
-  Listener or on the Caller. The value set on the Listener will win, if set on
-  both parties.
+  Listener (the service defines the encryption rules strictly) or on the Caller
+  (the service allows all clients to freely decide about encryption, or the
+  server uses more elaborate rules for encryption).
 
-  - **Bidirectional in Rendezvous arrangement**: you have to know the passphrases
-  for both parties, as well as `PBKEYLEN`. Set `PBKEYLEN` to the same value on
-  both parties (or leave the default value on both parties, which will
-  result in 16)
+  - **Bidirectional in Rendezvous arrangement**: In rendezvous there's no
+  situation of a potential "unknown party", as it is with listener side, so
+  both parties simply should set the same passphrase and the same key length,
+  or both should leave the `SRTO_PBKEYLEN` option unchanged (which results in
+  default 16).
```

Comment:
```suggestion
  - **Bidirectional in Rendezvous mode**: In Rendezvous mode it is not possible for a party to be "unknown", 
  as it is with Listener mode, so both parties should set the same passphrase and the same key length, 
  or both should leave the `SRTO_PBKEYLEN` option unchanged (which results in default 16).
```

Reply:
@ethouris I prefer the term "mode" to "arrangement" when writing about Caller, Listener and Rendezvous. However, this is subjective. If we use "mode", then it should replace "arrangement" throughout the document.

Reply:
So far the practice was that "mode" was either of: Listener, Caller, Rendezvous - that is, touches upon only particular party. "Arrangement" was how both parties are organized together in order to form a connection - hence "Caller-Listener" or "Rendezvou" was an "arrangement".